lib/msal-core/docs/account-object.md

```diff
@@ -0,0 +1,28 @@
+
+# Accounts in MSAL
+
+The MSAL SDKs have supported the notion of authenticating multiple user accounts. To align with this model, MSAL.js 1.x uses **Account** object. The object will surface the information about the authenticated user account.
```

Comment:
Do we want to be bolding **Account** (and other similar references to ideas in the code) or inline coding them like `this`?

Reply:
I will change it to the inline code format